### ISSUE TITLE
Revamp SSH provisioning section in the documentation

### ISSUE THREAD

**Issue by charlesprost** [scality/metalk8s#1219]
**Component**:

documentation

**Why this is needed**:

Because the current documentation is not working "out of the box" and is not really clear about how to test it and which user we should use.

**What should be done**:

* Add `-f` in the `ssh-copy-id` commandline, because since we don't copy the private key to our host, the proposed command is not working
* Explain which command is to be run from the bootstrap node to the new node to check if the SSH connection is working
* Add a note about the user that should be SSH ready; do not need to be `root` because we can use `sudo` capabilities while adding a new node to the cluster

**Test plan**:

* Run an extension with the new version of the documentation

**Comment by gdemonet** [comment]
This procedure also needs to be revamped to take into account _usual_ infrastructures: most of the time, SSH access from what we call the _host_ will only exist for the Bootstrap node. Then, other nodes can be reached from this Bootstrap node, with password- or public key-based authentication.

**Comment by charlesprost** [comment]
Yeah totally agree. Though currently we offer this one. Let's make it a suggestion and change the few things I'm suggesting.
